src/System.Collections.Concurrent/src/System/Collections/Concurrent/ConcurrentDictionary.cs

```diff
@@ -2058,12 +2057,12 @@ public object Key
                 get { return _enumerator.Current.Key; }
             }
 
-            public object Value
+            public object? Value
             {
                 get { return _enumerator.Current.Value; }
             }
 
-            public object Current
+            public object? Current
```

Comment:
entry is non-nullable, so this presumably also isn't

Reply:
Changed.